backend/entityservice/tasks/comparing.py

```diff
@@ -61,9 +61,8 @@ def create_comparison_jobs(project_id, run_id, parent_span=None):
         current_span.log_kv({"event": 'get-dataset-sizes'})
 
         filters_object_filenames = tuple(
-            get_filter_metadata(conn, dp_id) for dp_id in dp_ids)
+            get_filter_metadata(conn, dp_id)[0] for dp_id in dp_ids)
```

Comment:
Might want to keep and use the  entity id indicies that `get_filter_metadata` is now returning?

Reply:
`get_filter_metadata` returns the filename and the encoding size of a dp's CLKs.

Reply:
carry on then, nothing to see here